www/app/ga-tracking.js

```diff
@@ -1,13 +1,16 @@
 import ReactGA from 'react-ga';
+import getConfig from 'next/config';
+
+const { GA_TRACKING_ID } = getConfig().publicRuntimeConfig;
```

Comment:
Shouldn't we use `getConfig()` inside the script, making it return the updated value instead of running it when the file is loaded?